examples/calling-apis/spa-with-backend-api/create-agent-chat-app/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "@tsconfig/recommended/tsconfig.json",
+  "compilerOptions": {
+    "lib": ["ES2022", "DOM", "DOM.Iterable"],
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "module": "esnext",
+    "moduleDetection": "force",
+    "moduleResolution": "bundler"
```

Comment:
important callout here - had some issues w/ `NodeNext` resolution w/ the setup here. Although we do ship ESM and CJS w/ our packages now, the langgraph-cli seems to have trouble resolving packages at times w/ ESM and `NodeNext` resolution. Given this is an example, i think we are ok. But `NodeNext` can defiintely still be used (although it does impact import syntax / file extension syntax)

Reply:
captured for future as part of AIDX-186 (low priority)